lib/flag-filter.js

```diff
@@ -184,11 +184,15 @@ FlagFilter.prototype._transform = function (vm, enc, cb) {
 		vm.ptrname = vm.tags[DOCKER_PREFIX + INST_PTR_TAG];
 	}
 
-	/* And enable VMs owned by admin that have a service attached. */
-	if (vm.owner.login === 'admin' && vm.services.length > 0) {
+	/* 
+	 * And enable VMs owned by admin or poseidon that have a service 
+	 * attached. 
+	 */
+	if ((vm.owner.login === 'admin' || vm.owner.login === 'poseidon')
```

Comment:
I think we should keep the original `&& vm.services.length > 0` here. There's not a lot of point in generating instance records for everything in manta when we have the `MANTA_ROLE_WHITELIST` further up only generating services for `loadbalancer`.

We should either revisit that whitelist and add more services, or we should keep the original `&& vm.services.length > 0` here.